backend/src/services/worker/worker.service.ts

```diff
@@ -104,14 +95,31 @@ class Worker {
       passwordSalt,
     );
 
+    const workerByName = await this.#workerRepository.getByName(
+      name,
+      master.tenantId,
+    );
+
+    if (workerByName) {
+      throw new InvalidCredentialsError({
+        status: HttpCode.BAD_REQUEST,
+        message: `Worker with name ${name} exist`,
+      });
+    }
```

Comment:
```suggestion
    const workerByName = await this.#workerRepository.getByName(
      name,
      master.tenantId,
    );

    if (workerByName) {
      throw new InvalidCredentialsError({
        status: HttpCode.BAD_REQUEST,
        message: `Worker with name this name is exist`, 
      });
    }
```
use enum with messages instead of string (we already have this enum)